kolibri/utils/i18n.py

```diff
@@ -33,7 +33,7 @@ def get_installed_app_locale_path(appname):
         return module_locale_path
 
 
-def get_supported_languages(kolibri_module_path):
+def get_supported_language_info(kolibri_module_path):
```

Comment:
We can probably refactor this function to not need the argument, and just use the relative path (there's no reason that the file should move relative to this).

```suggestion
def get_supported_language_info():
    """
    Returns a list of dicts with all the language information
    """
    file_path = os.path.join('..', "locale", "supported_languages.json")
    with io.open(file_path, encoding="utf-8") as f:
        return json.load(f)

KOLIBRI_SUPPORTED_LANGUAGES = get_supported_language_info()
```

Reply:
Yeah I agree.

I've been blindly following how it was originally implemented - it looked weird but I assumed there was a reason for it so I didn't want to refactor... If you think it's safe to update I'll do that

Reply:
The only reason it was like this was because we were not using the information in the json file in any way except to generate the Django `LANGUAGES` setting previously. Seems safe to me!